tests/users/test_signup_view.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+
+from users.forms import SignUpForm
+
+
+class AccountsTestCase(TestCase):
+
+    def setUp(self):
+        self.user_data = {
+            'username': 'testuser124',
+            'first_name': 'test',
+            'last_name': 'demo',
+            'email': 'testuser@gmail.com',
+            'password1': '1234aditya_is_best',
+            'password2': '1234aditya_is_best'
+        }
+
+    def test_signup_page_elements_render(self):
+        """
+        Test for all the elements in the signup page is loding properly or not.
```

Comment:
Test whether all the elements in the signup view is loading properly or not.

Reply:
It's loading not loding. Check the PR yourself once and fix these typos before submitting PR.